src/FakeItEasy/Configuration/IRepeatConfiguration.cs

```diff
@@ -3,12 +3,14 @@ namespace FakeItEasy.Configuration
     /// <summary>
     /// Provides configuration for method calls that has a return value.
     /// </summary>
-    public interface IRepeatConfiguration : IHideObjectMembers
+    /// <typeparam name="TInterface">The type of configuration interface to return.</typeparam>
+    public interface IRepeatConfiguration<out TInterface> : IHideObjectMembers
     {
         /// <summary>
         /// Specifies the number of times for the configured event.
         /// </summary>
         /// <param name="numberOfTimesToRepeat">The number of times to repeat.</param>
-        void NumberOfTimes(int numberOfTimesToRepeat);
+        /// <returns>A configuration object that lets you define the following behavior.</returns>

```

Comment:
It's probably just me, but when I read "the following behavior", I expected to see the behavior on the next line or so. I wonder if maybe we shouldn't say "…define subsequent behavior for the method call"?